src/util/sys/mod.rs

```diff
@@ -60,7 +60,7 @@ pub mod pri {
 }
 
 #[cfg(not(target_os = "linux"))]
-pub mod pri {
+pub mod process_thread {
```

Comment:
It should be `thread` too, otherwise it wont compile on platforms other than `linux`.

Reply:
+1